### DISCUSSION TITLE
Erreur sur controle IP

### DISCUSSION THREAD

**Discussion by moiiom59** [dbarzin/mercator#514] [General]
Une erreur apparaît a l'enregistrement d'un serveur  physique lors d'une saisie d'Ip au format 128.001.061.150 pourriez vous corriger ?
merci

**Comment by dbarzin** [comment]
Le format de l'adresse IP est incorrect. Le bon format est 128.1.61.150

**Comment by moiiom59** [comment]
C’est problématique car ma solution d’inventaire sur laquelle je m’appui pour alimenter Mercator est au format avec les 0 significatif ! ne pourrait on pas accepter ce format ?

Bien cordialement,

**Comment by dbarzin** [comment]
Le format va être changé. Ce sera dispo lundi matin.

**Comment by dbarzin** [comment]
C'est disponible !

**Comment by moiiom59** [comment]
Merci à vous pour votre réactivité

Cordialement Laurent BASSET
RSSI CH DUNKERQUE
________________________________
From: Didier Barzin ***@***.***>
Sent: Sunday, October 1, 2023 1:24:01 PM
To: dbarzin/mercator ***@***.***>
Cc: moiiom59 ***@***.***>; Author ***@***.***>
Subject: [*Newsletter*] Re: [dbarzin/mercator] Erreur sur controle IP (Discussion #514)


C'est disponible !
